.env.example

```diff
@@ -36,3 +36,11 @@ RGDPS_DIR=./rgdps_data
 
 # RealistikGDPS Server Configuration
 SRV_NAME=RealistikGDPS
+
+# Logging Configuration
+LOG_LEVEL=INFO
+LOGZIO_ENABLED=0
```

Comment:
make it proper bool and then parse `val == "true"` for config readibility?

Reply:
Imo, this isnt the PR for this sort of change? The current config parser would support that though.